sqs/Module.md

```diff
@@ -1,83 +1,25 @@
-[![Build Status](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs)
+## Overview
+Ballerina connector for Amazon SQS is connecting the Amazon SQS API via Ballerina language easily. It provides capability to perform operations related to queues and messages.
```

Comment:
Change "is connecting" to "connects" and "easily" to 'with ease"

Reply:
Fixed